src/lib/chart/dendrogram.typ

```diff
@@ -219,12 +211,16 @@
   }
 
   let draw-data = (
-    if mode == "vertical" {vertical-draw-dendrogram} else
-    if mode == "horizontal" {horizontal-draw-dendrogram}
+    if mode == "vertical" {basic-draw-dendrogram} else
+    if mode == "horizontal" {basic-draw-dendrogram}
   )
```

Comment:
this isn't needed anymore

Reply:
Working on radial display now, better to leave this code in so it's there when its needed